source/containers.tex

```diff
@@ -2794,6 +2843,17 @@
     \bigoh{\tcode{b.size()}}.
 \\ \rowsep
 %
+\tcode{b.equal_range(k, hk)}
+&   \tcode{pair<iterator, iterator>}; \br
+    \tcode{pair<const_iterator, const_iterator>} for const \tcode{b}.
+&   \expects \tcode{b.hash_function()(k)} equals \tcode{hk}.\br
+    \returns a range containing all elements with keys equivalent to
+    \tcode{k}.  Returns \tcode{make_pair(b.end(), b.end())} if
+    no such elements exist.%
+&   Average case \bigoh{\tcode{b.count(k)}}.  Worst case
```

Comment:
We're inconsitent as to wether we do 
`Average case x.  Worst case y.`
vs
`Average case x, worst case y.`
Should we open an editorial issue for this?

Reply:
I would even vouch for a semicolon here. Yes, please make that a separate editorial issue and put "decision-required" on it.

Reply:
Opened #2749.